addon/-private/system/relationships/ext.js

```diff
@@ -12,8 +12,8 @@ var Map = Ember.Map;
 var MapWithDefault = Ember.MapWithDefault;
 
 var relationshipsDescriptor = Ember.computed(function() {
-  if (Ember.testing === true && relationshipsDescriptor._cacheable === true) {
-    relationshipsDescriptor._cacheable = false;
+  if (Ember.testing === true && relationshipsDescriptor._volatile === true) {
```

Comment:
It looks like updating the tests as well helps with a happy Travis :wink: Thanks again for your :eyes:!